src/Symfony/Component/Validator/Constraints/BicValidator.php

```diff
@@ -69,13 +68,16 @@ public function validate($value, Constraint $constraint)
             return;
         }
 
+        // @deprecated since Symfony 4.2, will throw a LogicException in 5.0
         if (!class_exists(Intl::class)) {
-            throw new LogicException('The "symfony/intl" component is required to use the Bic constraint.');
+            @trigger_error('Using the Bic constraint without the "symfony/intl" component installed is deprecated since Symfony 4.2.', E_USER_DEPRECATED);
+            $invalidCountryCode = !ctype_alpha(substr($canonicalize, 4, 2));
+        } else {
+            $invalidCountryCode = !isset(Intl::getRegionBundle()->getCountryNames()[substr($canonicalize, 4, 2)]);
         }
 
-        // next 2 letters must be alphabetic (country code)
-        $countries = Intl::getRegionBundle()->getCountryNames();
-        if (!isset($countries[substr($canonicalize, 4, 2)])) {
+        // next 2 letters must be a valid country code
```

Comment:
this comment looks a bit off now :) i'd drop it as the variable is self-descriptive now